libs/application-generic/src/usecases/get-subscriber-preference/get-subscriber-preference.usecase.ts

```diff
@@ -50,6 +50,7 @@ export class GetSubscriberPreference {
       },
     );
 
+    // TODO: replace this runtime mapping with a single query to the database
```

Comment:
This was the problematic piece of code that is now avoided in the `SendMessage` use-case.